### DISCUSSION TITLE
Best way to organise HIFIS tasks

### DISCUSSION THREAD

**Discussion by cmeessen** [research-software-directory/RSD-as-a-service#180] [General]
I just had a look at the boards functionality and was wondering how we would best organise tasks that belong to HIFIS. With the current solution where we have the Helmholtz view in the board, we would have to create a new HIFIS or Helmholtz label. This way we can use a label filter in the view to sort out all other issues. Is there another way to filter the view that I did not find?

The other option that comes to my mind is to create a separate HIFIS board, so issues would be added to this project.

Both approaches have their advantages. What would you recommend?

cc @jmaassen @ewan-escience @dmijatovic

**Comment by ewan-escience** [comment]
I think the best way is to open those tasks as issues in https://github.com/research-software-directory/Helmholtz and they should then be visible in https://github.com/orgs/research-software-directory/projects/5/views/10.
